File: Python/1038 Lanche.py
s = list(map(int, input().split()))
a = s[0]
x = 0.00
if a == 1:
  x = s[1]*4.0
elif a == 2:
  x = s[1]*4.5
elif a == 3:
  x = s[1]*5.0
elif a == 4:
  x = s[1]*2.0
elif a == 5:
  x = s[1]*1.5
print("Total: R$ {:.2f}".format(x))

File: C/1038 Lanche.c
#include <stdio.h>
 
int main() {
 int a, b;
 scanf("%d %d", &a, &b);
 
    double x = 0;

    switch (a)
    {
      case 1:
      x = b*4.0;
      printf("Total: R$ %.2lf\n", x);
      break;
      case 2:
      x = b*4.50;
      printf("Total: R$ %.2lf\n", x);
      break;
      case 3:
      x = b*5.0;
      printf("Total: R$ %.2lf\n", x);
      break;
      case 4:
      x = b*2.0;
      printf("Total: R$ %.2lf\n", x);
      break;
      case 5:
      x = b*1.50;
      printf("Total: R$ %.2lf\n", x);
      break;
    }
}